layers/+window-management/exwm/packages.el

```diff
@@ -31,12 +31,17 @@
       (exwm :location (recipe :fetcher github
                               :repo "ch11ng/exwm")
             :step pre)
+      (xdg :location local)
```

Comment:
local?

Reply:
Yeah, I'm not so sure about best practices here.
`xdg.el` is included in emacs.  To take advantage of the defer (if we even need to?) I've put it on the list of packages.  Is there better practice for this?  Would a simple `require` be better?

Reply:
`:location built-in`